### PR TITLE
updates to SMP March 14 relnotes + PR1000 + PR1002 DOC-2765

### PR DESCRIPTION
**Pull request by douglas-j-bothwell**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by akshay-khole — DISMISSED** [review]

**Review by pratmit — APPROVED** [review]
LGTM

File: release-notes/self-managed-enterprise-edition.md [hunk 1]
```diff
@@ -29,6 +29,80 @@ This release includes the following Harness module and component versions.
...
+
+    2. Proceed to upgrade timescaleDB using the installation methods (KOTS/Helm).
+
+    3. After the timescale pods are in steady state, run:
+
+            ALTER EXTENSION timescaledb UPDATE;
+
+    4. Validate the change by running:
+
+          \dx timescaledb
```
Comment (aimurphy) [inline release-notes/self-managed-enterprise-edition.md:94]: should these commands be wrapped like code?

File: docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md [hunk 2]
```diff
@@ -32,6 +32,56 @@ This release includes the following module and component versions.
 
 #### New features and enhancements
 
+- The **mongoDB minor version** is upgraded from 4.2 to 4.4. (SMP-837)
+
+  To upgrade to 4.4, do the following.
+
+  1. Get the current featureCompatibiltyVersion. It should be 4.2 if it is using mongo 4.2:
+
+          db.adminCommand( { getParameter: 1, featureCompatibilityVersion: 1 } )
```
Comment (aimurphy) [inline docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md:41]: same question here about code notation

**Review by aimurphy — APPROVED** [review]
Approved, just want to make sure if those commands are supposed to be marked up as code or not.

File: docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md (hunk 2 above)
Comment (douglas-j-bothwell) [reply docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md:41]: Yes, I formatted them as code.

**Pull request merged**